apps/frontend/src/components/WorkoutMenu/WorkoutOverview.tsx

```diff
@@ -147,3 +155,43 @@ export const WorkoutOverview = ({
     </Stack>
   );
 };
+
+type EditCurrentWorkoutButtonProps = {
+  currentWorkout: Workout | null;
+  allUserWorkouts: WorkoutWithType[];
+  previewFtp: number;
+  setWorkoutToEdit: (workout: WorkoutToEdit) => void;
+};
+
+const EditCurrentWorkoutButton = (props: EditCurrentWorkoutButtonProps) => {
+  const { currentWorkout, allUserWorkouts, setWorkoutToEdit, previewFtp } =
+    props;
+
+  if (!currentWorkout) return null;
+  const currentWorkoutType =
+    currentWorkout?.id === ''
+      ? 'new'
+      : allUserWorkouts.find(
+          (userWorkout) => userWorkout.workout.id === currentWorkout?.id
+        )?.type;
+  if (!currentWorkoutType) return null;
+
+  return (
+    <Button
+      textColor={'orange'}
```

Comment:
If we want it to be an other color, we should use the [`colorScheme` prop](https://v2.chakra-ui.com/docs/components/button/usage#button-colors). I don't think it is necessary in this case though.
```suggestion
```